py/client/examples/import_test_data.py

```diff
@@ -1,37 +1,39 @@
 #
 #  Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
 #
-""" Demo the steps to upload CSV data into Deephaven
-
+""" Demo the steps to upload Parquet data into Deephaven
     1. use pyarrow to load external data of interest into memory
     2. prepare the data
     3. import the data into the Deephaven server
-
 """
 
-from pyarrow import csv
+from pyarrow import parquet
 
-from examples.csv_downloader import download_csv
+from examples.file_downloader import download_file
 from pydeephaven import Session, Table
 from pydeephaven.utils import is_deephaven_compatible
 
 
 def import_taxi_records(dh_session: Session) -> Table:
 
-    # download the CSV data and read it into a pyarrow table and prepare it for uploading into DH
-    csv_file_name = download_csv(url="https://nyc-tlc.s3.amazonaws.com/trip+data/yellow_tripdata_2020-12.csv")
-    pa_table = csv.read_csv(csv_file_name)
+    # download the data and read it into a pyarrow table and prepare it for uploading into DH
+    file_name = download_file(url="https://nyc-tlc.s3.amazonaws.com/trip+data/yellow_tripdata_2020-12.parquet", file_name="sample.parquet")
+    pa_table = parquet.read_table(file_name)
 
     # drop unwanted columns
     unwanted_columns = ["tpep_pickup_datetime", "tpep_dropoff_datetime", "RatecodeID",
                         "store_and_fwd_flag", "PULocationID", "DOLocationID"]
     pa_table = pa_table.drop(unwanted_columns)
 
+    # this is necessary because otherwise the data is too big to be sent
+    new_len = pa_table.num_rows // 2
+    pa_table = pa_table.slice(length=new_len)
```

Comment:
After looking, I'm not sure what this means.  What makes it too big?  Out of RAM on the server, or something else?

Reply:
@SuperTails @jakemulf

Reply:
The table is too big to be sent as a gRPC message. The deephaven server errors out with the following:
`gRPC message exceeds maximum size 104857600: 109183697`

Reply:
@niloc132 making sure you've seen this.

Reply:
Short version, we want to have a max size of some kind - default was 5mb, we raised it to 100mb.

This is configurable, the property is `grpc.maxInboundMessageSize`, set to a value in bytes. This can be set from a prop file or additional args when starting the server.

Another resolution could be for the py client to break up its upload to smaller pieces, rather than send it all at once.